ci: add myself to the `renovate-changes` PullApprove group

alan-agius4 opened the pull request:
I frequently need to ping others to approve Renovate PRs, so this should streamline the process.

josephperrott commented on .pullapprove.yml:
```diff
@@ -684,6 +684,8 @@ groups:
       - author in ["angular-robot"]
     reviewers:
+      users:
+        - alan-agius4
```
```suggestion
        - ~alan-agius4
```

Can we switch to this so that you don't get them assigned as well?

josephperrott reviewed (approved):
LGTM

alan-agius4 committed:
ci: add myself to the `renovate-changes` PullApprove group

I frequently need to ping others to approve Renovate PRs, so this should streamline the process.

thePunderWoman commented:
This PR was merged into the repository by commit 55c442033f45127128c8228aa8f929aee2f8bee8.

The changes were merged into the following branches: main, 19.0.x
